clients/python/test/e2e/ci/check_for_failures.py

```diff
@@ -3,15 +3,25 @@
 import pandas as pd
 import pytest
 import typer
+from _data_classes import Artifacts, PytestIniFile
 from _utils import E2eExitCodes
+from pydantic import ValidationError
 
 
-def main(e2e_artifacts_dir: str):
+def main():
     """Loop through all json artifacts and fail in case of testfailure"""
-    artifacts = Path(e2e_artifacts_dir)
-    if not artifacts.is_dir():
-        typer.Exit(code=E2eExitCodes.CI_SCRIPT_FAILURE)
-    for pth in Path(artifacts).glob("*.json"):
+    try:
+        pytest_ini: PytestIniFile = PytestIniFile.read()
+    except (ValueError, ValidationError):
+        raise typer.Exit(code=E2eExitCodes.CI_SCRIPT_FAILURE)
```

Comment:
will this generate a "nice" log error in the CI? my last time looking at errors was a bit difficult. Things like:
- which deploy is causing the issue
was not quite straightforward to find out.

Reply:
Yep, I agree. I was really struggling to remove the traceback from the pytest log for gitlab ci. I could do so locally but didn't manage to do it in gitlab (big mystery!).
Hence, I decided to "mute" pytest completely and only print the configuration (client, server) and the return code. Then the user will have to look at the html produced by `pytest-html`, which is quite nice. In particular it allows to filter the passed/skipped/failed tests. 
Here is what the log in gitlab ci looks like now:
```
=============================================================================================
Configuration
-------------
	client: latest
	server: https://api.osparc-master.speag.com/
Exit status
-------------
	OK
=============================================================================================
=============================================================================================
Configuration
-------------
	client: latest
	server: https://api.osparc.io/
Exit status
-------------
	OK
=============================================================================================
=============================================================================================
Configuration
-------------
	client: latest
	server: https://api.osparc.speag.com/
Exit status
-------------
	TESTS_FAILED
=============================================================================================
=============================================================================================
Configuration
-------------
	client: latest
	server: https://api.osparc-staging.speag.com/
Exit status
-------------
	OK
=============================================================================================
=============================================================================================
Configuration
-------------
	client: latest
	server: https://api.staging.osparc.io/
Exit status
-------------
	OK
=============================================================================================
```